packages/forc-bin/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "forc-bin",
-  "version": "0.26.1",
+  "version": "0.30.0",
```

Comment:
No need to change the `version` field, only the `forcVesion`.

 - https://github.com/FuelLabs/fuels-ts/pull/563#discussion_r1010480647